### PR TITLE
Enhanced ends-on section

### PR DESCRIPTION
**Pull request by ojas-elawadhi**
- [x] Default ends on date should be 7 days
 - [x] Move number of days remaining on Ends on line for eg: Ends on:* 2021-11-26 7 days remaining
 - [x] Once date has been set date picker, input box must be hidden if date is valid
 - [x] Remove all commented lines in setEndDate function
 - [x] Make 5px Clickable Region Around Edit

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -207,8 +207,8 @@ footer {
 #remainingDays {
   /* all:unset; */
   float: left;
-  margin-block: -0.5em 0.75em;
-  padding-left: 0.5em;
+  margin-block: -3.9em 0.75em;
+  padding-left: 12.5em;
```
Comment (lakshayman) [inline main.css:211]: Why are you giving margins to the remaining days? Can't you place the remaining days in the label?

**Review by lakshayman — CHANGES_REQUESTED** [review]
Looks good to me

File: script.js [hunk 2]
```diff
@@ -288,3 +282,13 @@ edits.forEach((edit, index) => {
   });
   edit.append(element);
 });
+
+function handleDateChange(event) {
+  const preview = event.target.previousElementSibling.children[1];
```
Comment (lakshayman) [inline script.js:287]: Place const input = event.target in starting of the function
and make it

```suggestion
  const preview = input.previousElementSibling.children[1];
```

File: script.js [hunk 3]
```diff
@@ -288,3 +282,13 @@ edits.forEach((edit, index) => {
...
+  const edit = event.target.previousElementSibling.children[2];
```
Comment (lakshayman) [inline script.js:288]: This one too
```suggestion
  const edit = input.previousElementSibling.children[2];
```

**Review by Pavangbhat — CHANGES_REQUESTED** [review]
image
- [ ] By default ends on should be 7 days
image
- [ ] We need to have some margin here

File: main.css (hunk 1 above)
Comment (ojas-elawadhi) [reply main.css:211]: Yes I can place it in label but still I will need to give margins and change some properties to it to place it to correct position.
Please tell which is more correct way? I will do it that way...

File: script.js [hunk 4]
```diff
@@ -45,28 +45,24 @@ const isNoteworthy = document.getElementById('isNoteworthy');
...
     endDate.parentElement.querySelector('em').innerHTML = `${
       endTimeArray[2]
     }-${endTimeArray[0].padStart(2, '0')}-${endTimeArray[1].padStart(2, '0')}`;
 
-  const remainingDays = document.getElementById('remainingDays').children[0];
-  remainingDays.innerHTML = Math.ceil(
-    (endTime - startTime) / (1000 * 60 * 60 * 24),
-  );
-};
+/*   const remainingDays = document.getElementById('remainingDays').children[0];*/
```
Comment (Pavangbhat) [inline script.js:61]: Please remove the comments

**Review by Pavangbhat — COMMENTED** [review]
Tested on firefox  looks like it needs some margin 
image

File: script.js (hunk 4 above)
Comment (ojas-elawadhi) [reply script.js:61]: Ok will do the changes....but now it is showing correct default remaining days (7) on your side?

Comment (Pavangbhat) [reply script.js:61]: Yes Ojas

**Review by Pavangbhat — CHANGES_REQUESTED** [review]
Please make it responsive as well

image
Other than that everything looks good to me

**Pull request merged**